config/template/import-config.example.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+#
+# An example config file for handling imports, regardless of import method:
+#
+#  * import-wikis.sh: Importing wikis from local files
+#  * import-remote-wikis: A wrapper on import-wikis.sh which pulls wikis from
+#    remote servers via CIFS.
+
+#
+# GENERAL
+# These settings apply in all cases
+#
+
+# MySQL root password for local database
+mysql_root_pass="mypassword"
+
+# Use a webhook, or just put "n" for no
+slackwebhook="https://hooks.slack.com/your-slack-webhook"
+
+# space-delimited list of wikis you want to import.
+# If you want to import all wikis, do `which_wikis="IMPORT_ALL"`
+# This may or may not be used in multiple scripts
+which_wikis="wiki1 wiki2 wiki3"
+
+# path to your wiki imports
```

Comment:
I think this needs more explanation. Something explaining this dir is where backup files will be saved and then used for import. If this dir doesn't exist, will the script create the dir for me?